CommonLib/Error/GREYError+Private.h

```diff
@@ -60,9 +60,9 @@ GREY_EXTERN NSString *const kErrorFunctionNameKey;
 GREY_EXTERN NSString *const kErrorUserInfoKey;
 
 /**
- * Key used to retrieve the bundle ID from an error object dictionary.
+ * Key used to retrieve the error info dictionary from an error object dictionary.
  */
-GREY_EXTERN NSString *const kErrorBundleIDKey;
+GREY_EXTERN NSString *const kErrorErrorInfoKey;
```

Comment:
Is this key used anywhere? I couldn't find it.

Reply:
It is - see GREYError.m:128 and GREYElementInteraction.m:818 for example

Reply:
kErrorBundleIDKey was not being used so i removed it.